CodeStarterCamp_Week2/main.swift

```diff
@@ -6,7 +6,38 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
 
-print("Hello, World!")
+// 로또생성함수
+func createLottoNumbers(_: (Int)) -> Set<Int> {
```

Comment:
파라미터를 사용하지 않는 상태인데 이렇게 작성해주신 이유가 있을까요?
```suggestion
func createLottoNumbers(_ number: Int) -> Set<Int> {
```
추가적으로  와일드 카드는 함수네이밍으로 충분한 설명히 가능할 때 전달인자 레이블로 사용하곤 합니다.
하지만 현재 상황에서는 파라미터로 받아도 그 파라미터의 네이밍이 존재하지 않아 사용을 할 수 가 없는 형태 겠네요.